src/diffpy/structure/__init__.py

```diff
@@ -66,7 +66,7 @@ def loadStructure(filename, fmt='auto', **kw):
     """
     from diffpy.structure.parsers import getParser
     p = getParser(fmt, **kw)
-    rv = p.parseFile(filename)
+    rv = p.parseFile(str(filename))
```

Comment:
Please revert, the filename argument should be a string as specified in the docstring.
If you need to add support for pathlib.Path types, it should go to a separate PR, 
let's keep this PR focused on one topic.

Reply:
No problem, reverted.